clion/src/main/kotlin/org/rust/clion/profiler/dtrace/RsDTraceConfigurationExtension.kt

```diff
@@ -82,13 +84,14 @@ class RsDTraceConfigurationExtension : CargoCommandConfigurationExtension() {
     companion object {
         private val PROFILER_RUNNER_IDS = listOf(RsProfilerRunner.RUNNER_ID, RsProfilerRunnerLegacy.RUNNER_ID)
         private const val DYLD_INSERT_LIBRARIES = "DYLD_INSERT_LIBRARIES"
-        private const val BUNDLED_STARTER_PATH = "profiler/macosx/libosx-starter.dylib"
-        private const val STARTER_PATH_PROPERTY = "clion.profiler.osx.starter.path"
-        private fun profilerStarterPath(): File {
-            System.getProperty(STARTER_PATH_PROPERTY)?.let {
-                return File(it) //develop
-            }
-            return File(PathManager.getLibPath(), BUNDLED_STARTER_PATH) //production
+
+        // BACKCOMPAT: 2020.3
+        private val BUILD_211: BuildNumber = BuildNumber.fromString("211")!!
+
+        private fun profilerStarterPath(): File = if (ApplicationInfo.getInstance().build >= BUILD_211) {
+            File(PathManager.getBinPath(), "profiler/mac/libosx-starter.dylib")
+        } else {
+            File(PathManager.getLibPath(), "profiler/macosx/libosx-starter.dylib")
```

Comment:
It would be great not to handle it by ourselves and use CLion method not to worry about it in the future.
Let's made the corresponding change in CLion codebase. Also, it can be done together with dropping `internal` modifier from `validateDTraceSettings` function